File: Linguagem C Exercicios/C-Condicional/MENOR DE TRES.c
#include <stdio.h>
#include <stdlib.h>

int main()
{

    int n1,n2,n3,menor = 0;

    printf("Primeiro valor: ");
    scanf("%d",&n1);
    printf("Segundo valor: ");
    scanf("%d",&n2);
    printf("Terceiro valor: ");
    scanf("%d",&n3);

    if (n1 < n2 && n1 < n3){
        menor = n1;
    }else if(n2 < n3){
        menor = n2;
    }else{
        menor = n3;
    };

    printf("MENOR = %d",menor);


    return 0;
}

File: Linguagem Java/Exercicios/MenorDeTres.java
import java.util.Locale;
import java.util.Scanner;

public class MenorDeTres {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int a,b,c,menor;
		
		System.out.print("Digite o primeiro valor: ");
		a = sc.nextInt();
		System.out.print("Digite o segundo valor: ");
		b = sc.nextInt();
		System.out.print("Digite o terceiro valor: ");
		c = sc.nextInt();
		
		if (a < b && a < c) {
			menor = a;			
		}else if (b < c) {
			menor = b;
			
		}else {
			menor = c;			
		};	
		
		System.out.println("Menor = "+menor);
		
		sc.close();
	}

}
